runtime/ext.rs

```diff
@@ -48,6 +48,8 @@ deno_core::extension!(
       "06_util.js",
       "90_zinnia_apis.js",
       "98_global_scope.js",
+      "internals.js",
+      "test.js",
```

Comment:
FWIW, we are always bundling and loading our test framework, even when running a module inside Filecoin Station via `zinniad`.

While it's not ideal, I think it's acceptable for the current state of our project.